image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -48,8 +48,10 @@ spec:
         env:
         - name: COLLECTOR_CONFIG
           value: '{"tlsConfig":{"caCertPath":"/var/run/secrets/stackrox.io/certs/ca.pem","clientCertPath":"/var/run/secrets/stackrox.io/certs/cert.pem","clientKeyPath":"/var/run/secrets/stackrox.io/certs/key.pem"}}'
+        {{- if (default ._rox.collector.forceCollectionMethod false) }}
```

Comment:
IIUC before your change `COLLECTION_METHOD` would always be set to some value. Either the default or whatever was provided in helm values.

When there's no `COLLECTION_METHOD` env var set at all, will collector use `CORE_BPF` by default or would it be better to explicitly set COLLECTION_METHOD to `CORE_BPF` in a else here?

Reply:
Yes, the collection used by default (no env var set) in collector has been set to CORE_BPF
https://github.com/stackrox/collector/commit/aea6d9d847fb66f7d2b8433ef06080da041b82f4#diff-e71abaf916cf53943f6a1094098e0000e72f7a06789de22e590aa88ad2388a29L23

But I agree that this is maybe a bold change. Would you prefer if there was an else block with an explicit COLLECTION_METHOD=CORE_BPF, @johannes94 ?

Reply:
I think yes, it would also be easier for customers to see which collection method is set, if they'd look into the DaemonSet or Pod and there is a env var.